readme.md

```diff
@@ -205,12 +205,13 @@ This table shows previous LTS releases and end dates.
 
 | Release                                                                                                                        | LTS Start Date | Maintenance End Date | LTS End Date |
 | :----------------------------------------------------------------------------------------------------------------------------: | :------------: | :------------------: | :----------: |
+| [2022-01-18](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2021-3-18-patch4) <sub>patch 4 of 2021-03-18</sub> | 2022-01-18     | 2022-03-18           | 2024-03-17   |
```

Comment:
Looks like we failed to add this to main after last adding to the latest LTS branch's readme, too.